command/alias.go

```diff
@@ -0,0 +1,105 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cli/cli/utils"
+	"github.com/google/shlex"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(aliasCmd)
+	aliasCmd.AddCommand(aliasSetCmd)
+}
+
+var aliasCmd = &cobra.Command{
+	Use:   "alias",
+	Short: "Create shortcuts for gh commands",
+}
+
+var aliasSetCmd = &cobra.Command{
+	Use: "set <alias> <expansion>",
+	// TODO HACK: even when inside of a single-quoted string, cobra was noticing and parsing any flags
+	// used in an alias expansion string argument. Since this command needs no flags, I disabled their
+	// parsing. If we ever want to add flags to alias set we'll have to figure this out. I haven't
+	// checked if this is fixed in a new cobra.
+	DisableFlagParsing: true,
+	Short:              "Create a shortcut for a gh command",
+	Long: `
+This command lets you write your own shortcuts for running gh. They can be simple strings or accept placeholder arguments.`,
```

Comment:
Nit: right now nothing strips the leading newline from the `Long` string, so the `gh help alias set` output starts with a blank line. Perhaps you could start the help text directly after `` Long: ` ``.

Also, it might be a nice touch to wrap any long help text lines to ~80 characters.

Reply:
The blank line was intentional because I thought it helped the output seem less crowded :shrug: i took it out